[2024-02-20T08:21:10.900+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 1Success_dag.select_opera manual__2024-02-20T00:00:00+00:00 [queued]>
[2024-02-20T08:21:10.909+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 1Success_dag.select_opera manual__2024-02-20T00:00:00+00:00 [queued]>
[2024-02-20T08:21:10.910+0000] {taskinstance.py:2170} INFO - Starting attempt 4 of 4
[2024-02-20T08:21:10.927+0000] {taskinstance.py:2191} INFO - Executing <Task(PostgresOperator): select_opera> on 2024-02-20 00:00:00+00:00
[2024-02-20T08:21:10.933+0000] {standard_task_runner.py:60} INFO - Started process 810 to run task
[2024-02-20T08:21:10.936+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '1Success_dag', 'select_opera', 'manual__2024-02-20T00:00:00+00:00', '--job-id', '12453', '--raw', '--subdir', 'DAGS_FOLDER/success_dag.py', '--cfg-path', '/tmp/tmpporp3olp']
[2024-02-20T08:21:10.938+0000] {standard_task_runner.py:88} INFO - Job 12453: Subtask select_opera
[2024-02-20T08:21:10.987+0000] {task_command.py:423} INFO - Running <TaskInstance: 1Success_dag.select_opera manual__2024-02-20T00:00:00+00:00 [running]> on host 06573eafe810
[2024-02-20T08:21:11.071+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='1Success_dag' AIRFLOW_CTX_TASK_ID='select_opera' AIRFLOW_CTX_EXECUTION_DATE='2024-02-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-20T00:00:00+00:00'
[2024-02-20T08:21:11.073+0000] {sql.py:276} INFO - Executing: 
        SELECT * FROM weather_tab
[2024-02-20T08:21:11.081+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-20T08:21:11.089+0000] {base.py:83} INFO - Using connection ID 'postgres_localhost' for task execution.
[2024-02-20T08:21:11.101+0000] {sql.py:450} INFO - Running statement: 
        SELECT * FROM weather_tab, parameters: None
[2024-02-20T08:21:11.107+0000] {sql.py:459} INFO - Rows affected: 419
[2024-02-20T08:21:11.142+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=1Success_dag, task_id=select_opera, execution_date=20240220T000000, start_date=20240220T082110, end_date=20240220T082111
[2024-02-20T08:21:11.189+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-20T08:21:11.204+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
